plugins/auth/auth_base.js

```diff
@@ -100,15 +103,19 @@ exports.check_user = function (next, connection, credentials, method) {
             connection.notes.auth_fails = 0;
         }
         connection.notes.auth_fails++;
+        connection.results.add(plugin, {fail: method});
 
         connection.notes.auth_login_userlogin = null;
         connection.notes.auth_login_asked_login = false;
 
         var delay = Math.pow(2, connection.notes.auth_fails - 1);
-        if (plugin.timeout && delay >= plugin.timeout) { delay = plugin.timeout - 1; }
-        connection.lognotice(plugin, 'delaying response for ' + delay + ' seconds');
+        if (plugin.timeout && delay >= plugin.timeout) {
+            delay = plugin.timeout - 1;
+        }
+        connection.lognotice(plugin, 'delaying for ' + delay + ' seconds');
```

Comment:
We could be more helpful if you tell us how you tried creating the PR and at which point [in the contributing process](https://github.com/baudehlo/Haraka/wiki/Contributing) you got stuck.